### ISSUE TITLE
BuffOverlay stopped working

### ISSUE THREAD

**Issue by F1NAM0RE** [clicketz/buff-overlay#18]
### Description

Addon doesn't appear in-game anymore after most recent update. 

### BuffOverlay Version

v12.6.20

### World of Warcraft Flavor

Vanilla Classic

### Sterile Testing

Yes

### Editing Lua Files

No

### Frames Used

EasyFrames

### Reproduction Steps

.

**Comment by clicketz** [comment]
Do you have "Load out of date AddOns" enabled? Are you trying to get it to appear on a particular addon that doesn't show anymore?

![JQstkmB](https://github.com/clicketz/buff-overlay/assets/40314230/cd01e883-d5db-4b03-bba6-3aae2725bdba)

Here is an image of it on my classic client with only BuffOverlay and EasyFrames enabled, so maybe I'm not fully understanding what your error is.

**Comment by F1NAM0RE** [comment]
That's exactly how my original blizzard raid frames looked prior to the recent update of Classic. Out of date addons are loaded. 

Logged in today and the addon said it was downloaded on Curseforge but not available in-game in any way, no command was responding and it wasn't available in the in-game addons list. I tried completely reinstalling it via Curseforge again but I can't get it to work.

**Comment by clicketz** [comment]
I just reinstalled from a fresh install on classic client and everything is working as intended (I installed it using WoWUP-CF though).

This might be a curseforge app issue?

Try downloading it directly from here: https://github.com/clicketz/buff-overlay/releases/download/v12.6.20/BuffOverlay-v12.6.20.zip

And place the `BuffOverlay` folder inside the zip file in `World of Warcraft\_classic_\Interface\AddOns` and let me know if that fixes it.

The final thing to try is to type this in-game: `/run BuffOverlayDB=nil;ReloadUI()` and see if that helps. (Note that this will erase any and all custom settings you created, unfortunately)
